lib/rouge/lexers/perl6.rb

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+
+module Rouge
+  module Lexers
+    class Perl6 < Perl
+      title "Perl 6"
+      desc "The Perl 6 programming language (perl6.org)"
+
+      tag "perl6"
+      aliases 'pl6'
```

Comment:
`aliases` are aliases for tags. I think it is not needed.

Reply:
Thanks, though I'm a bit confused here since the docstrings for `tag` and `aliases` are

```
Rouge::Lexer.tag([t]) Used to specify or get the canonical name of this lexer class.
Rouge::Lexer.aliases(args...) Used to specify alternate names this lexer class may be found by.